src/store/player.js

```diff
@@ -144,7 +144,10 @@ export default {
   },
   getters: {
     playlistTracks(state, getters, rootState) {
-      return state.playlist.map((id) => rootState.tracks.tracks[id]);
+      return state.playlist.reduce((acc, id) => {
+        const track = rootState.tracks.tracks[id];
+        return track ? [...acc, track] : acc;
+      }, []);
```

Comment:
This creates a lot of new arrays unnecessarily. A filter + map would probably be better.